### DISCUSSION TITLE
On-Site Set Backlog Feature

### DISCUSSION THREAD

**Discussion by pinguupinguu** [RetroAchievements/RAWeb#675] [Ideas]
An idea I think might be fun for users to have (looking toward V2 more-so for this) is a way to add a set to a personal list to play later.  I wonder that this could be added under the "My Pages" drop-down menu we have already on the site? I'd imagine it would only require the game ID to be displayed on the page, if not the point count too for extra fun. 

It's a feature that I would definitely be interested in using, as it would be an indirect way to recall games that interested me on the site currently!

**Comment by Jamiras** [comment]
There's already an issue for this: #799. I'm not sure how we're planning to use discussions - if that should be closed, or this should be.
